gradle.properties

```diff
@@ -1 +1,4 @@
-org.gradle.jvmargs=-Xmx2560M
+org.gradle.jvmargs=-Xmx1G -XX:SoftRefLRUPolicyMSPerMB=10 -Dlog4j2.disable.jmx=true -Dlog4j.shutdownHookEnabled=false
+org.gradle.parallel=true
+# In most instances the leaking of the Xmx worth of memory isn't worth the slight reduction in startup time
+org.gradle.daemon=false
```

Comment:
No, dont force this on people. Default should be off. And no point in disabling the deamon